src/dataflow/src/source/kinesis.rs

```diff
@@ -257,68 +344,23 @@ async fn create_state(
     let shard_set: HashSet<String> = get_shard_ids(&kinesis_client, &c.stream_name).await?;
     let mut shard_queue: VecDeque<(String, Option<String>)> = VecDeque::new();
     for shard_id in &shard_set {
+        let kinesis_id = PartitionId::Kinesis(shard_id.clone());
         shard_queue.push_back((
             shard_id.clone(),
             get_shard_iterator(&kinesis_client, &c.stream_name, shard_id).await?,
-        ))
+        ));
+        println!("Inserting {:?} ", kinesis_id);
```

Comment:
I'm not sure we need this println, but switch to logging if we do!

Reply:
Just forgot to remove the debug prints.